### PR TITLE
public getter getFocusRadius() & getFocusPosition

### PR DESCRIPTION
**Pull request by turbolay**
Hi, did issue #8, basically to test pull requests.
In hasRadius(), used equals.(CIRCLE) to be in correlation with other parts of the code, let redundancy to make it easier to understand in the case of new shapes. Didn't  find any other way to know if shape has a radius. Wanted firstly to use switch case, but then saw you're not using any in the whole code.

Hope that's what your expecting from a pull request like this.

### CODE REVIEW

**Comment by faruktoptas** [comment]
Please see my comment:
https://github.com/faruktoptas/FancyShowCaseView/issues/8#issuecomment-290308312

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java [hunk 1]
```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
         mDismissListener = dismissListener;
     }
 
+    /**
+     * Returns if current shape has a radius. Have to add more cases if new shapes doesn't have radius
+     *
+     * @return true if current shape has a radius
+     */
+    public boolean hasRadius(){
```
Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: This method is unnecessary. `getFocusRadius()` will return `-1` if there is no focus.

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java [hunk 2]
```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
...
+
+        return false;
+    }
+
+    /**
+     * Returns mRadius or -1 if mRadius == 0 (i.e. Radius doesn't exist for the shape)
+     *
+     * @return mRadius or -1 if mRadius == 0
+     */
+    public int getFocusRadius(){
```
Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Only returning `mRadius` will be enough.
Also `mRadius` initial value will be `-1`

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java [hunk 3]
```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
...
+        if(hasRadius())
+            return mRadius;
+        else
+            return -1;
+    }
+
+    public Point getFocusPosition(){
```
Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Point is never used here. You may only add these getters:
```
getFocusCenterX()
getFocusCenterY()
```

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java (hunk 2 above)
Comment (turbolay) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Are you sure about this? I've performed several tests on rounded rectangles, and everytime mRadius was == to 175. It's initialized in Calculator line 40 with height and width, enven for ROUNDED_RECTANGLE. So I think if getFocusRadius() has to return -1 for ROUNDED_RECTANGLE, Calculator has to be changed or a test has to be performed in getFocusRadius() no?
Btw ty for your time.

Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Line 40 runs if `view != null`. But view may be null.
I mean [here](https://github.com/faruktoptas/FancyShowCaseView/blob/master/library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java#L90) with initial value.

**Comment by faruktoptas** [comment]
Are you still working on this issue?

**Comment by faruktoptas** [comment]
This feature is added with [fbfe89d](https://github.com/faruktoptas/FancyShowCaseView/commit/fbfe89d70626d3bfb480e683caba74cfed926011)

**Pull request closed without merging**